lib/genKeys.js

```diff
@@ -1,16 +1,11 @@
 'use strict'
 
-function genKeys (obj, set) {
+function genKeys (obj) {
   return Object.keys(obj)
-    .filter(notInSet, set)
     .filter(noObjects, obj)
     .map(keyIterator, obj)
 }
 
-function notInSet (key) {
-  return !this || this.has(key)
-}
-
```

Comment:
That was introduced to solve https://github.com/mcollina/bloomrun/commit/5f41e75382bf185dd07d97743826e4fc487ede08.

That's why perf are dropping for 500 hundred entities benchmark.